packages/rx_bloc_cli/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: rx_bloc_cli
 description: "rx_bloc_cli that enables quick project setup including: flavors, localization [intl], state management [rx_bloc], routing [auto_route], design system, analytics [firebase], tests"
-version: 1.1.2
```

Comment:
I guess that this PR needs to be merged after https://github.com/Prime-Holding/rx_bloc/pull/277